src/components/VideoPlayer/BaseVideoPlayer.tsx

```diff
@@ -215,21 +215,24 @@ function BaseVideoPlayer({
         }
 
         // If we are uploading a new video, we want to immediately set the video player ref.
```

Comment:
Let's update this comment which also mentions pausing the current playing video.

Reply:
@Pujan92 
I've updated comment. Thanks.